tools/documentation.vnufilter

```diff
@@ -3,6 +3,7 @@
 Consider using the “h1” element as a top-level heading only \(all “h1” elements are treated as top-level headings by many screen readers and other tools\)\.
 Document uses the Unicode Private Use Area\(s\), which should not be used in publicly exchanged documents\. \(Charmod C073\)
 Section lacks heading\. Consider using “h2”-“h6” elements to add identifying headings to all sections, or else use a “div” element instead for any cases where no heading is needed\.
+Stray end tag “input”\.
```

Comment:
Has an upstream bug been filed for this?

Reply:
Not that I am aware of.